src/main/java/com/gd/clinic/entity/Prescription.java

```diff
@@ -0,0 +1,56 @@
+package com.gd.clinic.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.gd.clinic.model.NewPrescriptionDto;
+import lombok.*;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import java.time.OffsetDateTime;
+import java.util.*;
+
+@NoArgsConstructor
+@RequiredArgsConstructor
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(name = "prescriptions")
+public class Prescription {
+
+    @Id
+    @GeneratedValue(generator = "UUID")
+    @GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator")
+    @Type(type = "pg-uuid")
+    private UUID id;
+
+    @NonNull
+    @ManyToOne
+    @JoinColumn(name = "patient_id")
+    @JsonIgnoreProperties("prescriptionList")
```

Comment:
you will not need to return entities, for this you will have a separate object

Reply:
Yes, corrected this this commit.